server/graphql/mutations/orders.js

```diff
@@ -455,30 +455,36 @@ export async function addFundsToOrg(args, remoteUser) {
     models.Collective.findById(args.CollectiveId),
     models.Collective.findById(args.HostCollectiveId)
   ]);
-  let paymentMethod = await models.PaymentMethod.findOne({ where: {
+  // if a PaymentMethodId is defined it means we are going to try to
+  // increase the balance of an existing Payment method
+  if (args.PaymentMethodId) {
+    const paymentMethod = await models.PaymentMethod.findOne({ where: {
+      id: args.PaymentMethodId,
+    } });
+    // check whether query found the payment method id
+    if (paymentMethod) {
+      await paymentMethod.update({
+        initialBalance: paymentMethod.initialBalance + args.totalAmount,
+      });
+      return paymentMethod;
+    }
+  }
+  // otherwise, creates a new Payment method
+  const paymentMethod = await models.PaymentMethod.create({
+    name: args.description || 'Host funds',
+    initialBalance: args.totalAmount,
+    monthlyLimitPerMember: args.totalAmount,
```

Comment:
no reason to define the monthlyLimitPerMember to this value. I'd keep it to null.
(that's what created the issue for Pia if I understand correctly).